src/app/manager-dashboard/reports/reports.service.ts

```diff
@@ -219,4 +219,22 @@ export class ReportsService {
     });
   }
 
+  getCourseProgress({ planetCode, tillDate, fromMyPlanet, filterAdmin }: ActivityRequestObject = {}) {
+    return forkJoin([
```

Comment:
Use the methods from `CoursesService` instead, similar to `CertificationsViewComponent`